README.md

```diff
@@ -2,61 +2,116 @@
+- [`Xcode`](https://developer.apple.com/xcode/) (for iOS development)
 
-- Clone the project
+Once you have these installed, follow these steps:
 
-  `git clone https://github.com/scorelab/Go-social.git`
+1. Clone the repository to your machine:
 
-- Change the directory to the project folder
+   ```sh
+   git clone https://github.com/scorelab/Go-social.git
+   ```
 
-  `cd Go-social`
+2. Navigate to the project directory:
 
-- Install the node modules with the following command
+   ```sh
+   cd Go-social
+   ```
 
-  `npm install`
+3. Install all dependencies:
 
-### Configure the config.example.js file (IMPORTANT)
+   ```sh
+   npm install
+   # Or
+   yarn install
+   ```
 
-> Open the project in any text editor you use. Open the `config.example.js` file in the `config` folder and follow the following steps.
+Before using the app, you'll need to configure your app.
 
-- Use [this link](https://console.firebase.google.com/) to **create a Firebase** project for the Go-social application. It is free for everyone. Please use the web app instead of the Android or iOS app.
+## Go-social app Configuration ~[IMPORTANT!]
 
-- Update the **Sign-in method** of Firebase [project](https://firebase.google.com/docs/auth/android/password-auth#before_you_begin) that you want to use as **Enabled**
+To configure your app, follow these steps:
 
-- Create a **Firebase Database** and Update the Ruleset of the real-time database to **Read/Write Allow**.
+#### A. Firebase Setup:
 
-- Use [this link](https://cloud.google.com/maps-platform/) to **generate Google Map Api key** for the map view in G-social. This is not necessary. You can use the given api key. But it is not guaranteed that the given key will always work. It is better to have your own key. Enable all maps, routes, places in your key.
+1. Go to the [Firebase Console](https://console.firebase.google.com/) and create a new project for the Go-social application. This is free for everyone. It's recommended to use the **Web App** instead of the Android or iOS app.
 
-- Now you need a **Facebook app id** if you want to enable Facebook login for Go-social. Use [this link](https://developers.facebook.com/) for that and make sure to enable both email/password and Facebook sign-in method in Firebase.
+2. To enable `Authentication` and `Storage` for your app, activate them in the Firebase console `Build` section after creating and registering the app.
+3. Enable `Sign-in methods` Email/Password and Facebook in the Authentication section of your Firebase app.
+4. Create a `Firebase Realtime Database` and update its ruleset to allow read and write access.
+5. Update the Firebase configuration object in `config/config.js` with app-specific config details obtained from the Firebase console app settings.
 
-So place your Firebase details and Google map API key in **config.example.js** file and **rename it** to **config.js**.
```

Comment:
Any reason to remove this?

Reply:
I removed this because in the new version of React Native, Google Maps and Facebook SDK do not require passing the map_api_key and facebook_id separately. Instead, they should be configured in their respective files (Android and iOS). I have already added guidelines on how to configure them.